tools/applicationinsights-web-snippet/src/applicationinsights-web-snippet.ts

```diff
@@ -12,8 +13,11 @@ function webSnippetVersion() {
     return "";
 }
 
-function getSdkLoaderScript(config: SdkLoaderConfig) {
+function getSdkLoaderScript(config: SdkLoaderConfig, futureCfg?: ISnippetConfig) {
```

Comment:
Lets not do this, when we want to do this we will "update" the `SdkLoaderConfig` with the supported items